raspberry-pi/5/overlay.nix

```diff
@@ -0,0 +1,6 @@
+final: _prev: {
+  linux_rpi5 = final.linux_rpi4.override {
```

Comment:
Can we place remove the overlay here? The kernel can be also defined without it as this overlay doesn't actually overwrite an package. This is important for large installations where nixpkgs is shared between different nixos systems in fact if you set `nixpkgs.pkgs` than setting `nixpkgs.overlays` will not evaluate.

Reply:
There seemed to already be inconsistency in nixos-hardware between the two approaches. I arbitrarily chose to use an overlay like for `kobol/helios4`.

Reply:
I tried to remove it, where it was not needed, but maybe missed that case.

Reply:
This is now also part of: https://github.com/NixOS/nixos-hardware/pull/932